includes/widgets/table.php

```diff
@@ -5,64 +5,63 @@
 	exit; // Exit if accessed directly.
 }
 
-use Elementor\Core\Kits\Documents\Tabs\Global_Colors;
+
 use Elementor\Core\Kits\Documents\Tabs\Global_Typography;
 
 /**
- * Elementor testOne widget.
+ * Elementor table widget.
  *
- * Elementor widget that displays an eye-catching headlines.
  *
  * @since 1.0.0
  */
-class Widget_testOne extends Widget_Base {
+class Widget_Table extends Widget_Base {
 
 	/**
 	 * Get widget name.
 	 *
-	 * Retrieve testOne widget name.
+	 * Retrieve table widget name.
 	 *
 	 * @since 1.0.0
 	 * @access public
 	 *
 	 * @return string Widget name.
 	 */
 	public function get_name() {
-		return 'testOne';
+		return 'table';
 	}
 
 	/**
 	 * Get widget title.
 	 *
-	 * Retrieve testOne widget title.
+	 * Retrieve table widget title.
 	 *
 	 * @since 1.0.0
 	 * @access public
 	 *
 	 * @return string Widget title.
 	 */
 	public function get_title() {
-		return esc_html__( 'testOne', 'elementor' );
+		return esc_html__( 'table', 'elementor' );
```

Comment:
I think we always use capital letters here:

return esc_html__( 'Table', 'elementor' );

Reply:
I changed it